Sources/XiEditor/AppDelegate.swift

```diff
@@ -250,13 +258,32 @@ class AppDelegate: NSObject, NSApplicationDelegate {
                 return
             }
         }
+        informationalAlert(title: message, message: info)
+        renameCliToggle()
+    }
+    
+    func renameCliToggle() {
+        let destPath = URL(fileURLWithPath: "/usr/local/bin/xi")
+        if FileManager.default.fileExists(atPath: destPath.path) {
+            cliToggle.title = CliButtonState.remove.rawValue
+        } else {
+            cliToggle.title = CliButtonState.install.rawValue
+        }
+    }
+    
+    func informationalAlert(title: String, message: String) {
         let alert = NSAlert()
         alert.addButton(withTitle: "OK")
         alert.alertStyle = .informational
-        alert.messageText = message
-        alert.informativeText = info
+        alert.messageText = title
+        alert.informativeText = message
         alert.runModal()
     }
+    
+    enum CliButtonState: String {
+        case install = "Install CLI"
```

Comment:
I'm thinking about these names. I don't love including 'CLI' in the UI, but I'm not sure what I actually prefer.

'Install Command Line Tool'?
'Install `xi` Shell Command'?
'Install Command Line `xi`'?

(This is assuming we can use an `NSAttributedString` to use Monaco or some other system monospace font)

Reply:
My 👍 goes for:
> Install xi Shell Command

Reply:
Enums don't seem to play nice with `NSAttributedString` so I renamed it to `Install 'xi' Shell Command`.

Reply:
Okay, and it also probably doesn't play well with localization?

One way I would have also considered addressing this is to not use `rawValue` as the title; we could have a 'menuItemTitle: NSAttributedString { get }` calculated property on the enum, and we could identify the variant by having an `Int` raw value that we set as the `tag` of the `NSMenuItem`. But I think this is good, just mentioning how I would approach that if it were the goal.